syft/frameworks/torch/tensors/interpreters/native.py

```diff
@@ -915,6 +915,15 @@ def share(
             requires_grad (bool): Should we add AutogradTensor to allow gradient computation,
                 default is False.
         """
+        if crypto_provider is None:
+            raise ValueError(
```

Comment:
Actually can we raise a Warning instead?
I discussed with Andrew and we shouldn't modify the udacity tests, sorry :/ 
A warning should be a good in between